### DISCUSSION TITLE
Change icon for 'save' in Territory View

### DISCUSSION THREAD

**Discussion by aliya-ryan** [digidem/mapeo-desktop#600] [Ideas]
In MD Territory View

Currently the icon for save is an upload icon, as it used to be about uploading to OSM. It isn't very clear or obvious, and would make more sense if it was a little disk icon, as is usually used for save, and perhaps if it was a different colour so that it stood out more, as saving is important but it isn't obvious that you need to do it.

**Comment by emreiser** [comment]
I agree that this button needs to be far more prominent on the screen if it is a necessary step for saving data. However, I think ideally we would **remove this save step altogether.** Users have to accept changes to each element in the feature panel and I would vote for having these changes saved by default. I know we inherited save/upload from iD but are there reasons we can't eliminate it?
